### PR TITLE
#47 #67

### PR DESCRIPTION
**Pull request by lbarjak**

### CODE REVIEW

File: src/main/java/org/rulez/demokracia/pdengine/IVoteManager.java [hunk 1]
```diff
@@ -15,11 +15,14 @@ VoteAdminInfo createVote(String voteName, Set<String> neededAssurances, Set<Stri
 			boolean isPrivate, int minEndorsements) throws ReportedException;
 
 	Vote getVote(String voteId);
+
```
Comment (magwas) [inline src/main/java/org/rulez/demokracia/pdengine/IVoteManager.java]: We have enough empty lines, thx.

File: src/main/java/org/rulez/demokracia/pdengine/IVoteManager.java [hunk 2]
```diff
@@ -15,11 +15,14 @@ VoteAdminInfo createVote(String voteName, Set<String> neededAssurances, Set<Stri
...
 
 	String addChoice(String adminKey, String voteId, String choiceName, String user);
 
 	ChoiceEntity getChoice(String voteId, String choiceId);
 
-	void endorseChoice(String adminKey, String voteId, String choiceId, String string);
+	void endorseChoice(String adminKey, String voteId, String choiceId, String string) throws ReportedException;
+	void modifyVote(String adminKey, String voteId, String voteName);
 
 }
```
Comment (magwas) [inline src/main/java/org/rulez/demokracia/pdengine/IVoteManager.java]: A new line here, please.

**Review by magwas — APPROVED** [review]
Thank you, good work!

**Comment by lbarjak** [comment]
Nem zártam be, nem is tudom visszanyitni.

**Pull request closed without merging**